platform/Cargo.toml

```diff
@@ -12,4 +12,5 @@ dpe_profile_p256_sha256 = []
 dpe_profile_p384_sha384 = []
 
 [dependencies]
-openssl = {version = "0.10", optional = true}
\ No newline at end of file
+openssl = {version = "0.10", optional = true}
+ufmt = { git = "https://github.com/korran/ufmt.git", rev = "1d0743c1ffffc68bc05ca8eeb81c166192863f33", features = ["inline"] }
```

Comment:
fyi: I also had to add `ufmt` to dpe/Cargo.toml

Reply:
Done